packages-next/fields/src/types/relationship/views/cards/index.tsx

```diff
@@ -123,125 +133,150 @@ export function Cards({
 
   return (
     <Stack gap="xlarge">
-      {[...value.currentIds].map(id => {
-        const itemGetter = items[id];
-        return value.itemsBeingEdited.has(id) && onChange !== undefined ? (
-          <CardContainer mode="edit" key={id}>
-            <InlineEdit
-              list={foreignList}
-              fields={field.display.inlineEdit!.fields}
-              onSave={newItemGetter => {
-                setItems({
-                  ...items,
-                  [id]: newItemGetter,
-                });
-                const itemsBeingEdited = new Set(value.itemsBeingEdited);
-                itemsBeingEdited.delete(id);
-                onChange({
-                  ...value,
-                  itemsBeingEdited,
-                });
-              }}
-              selectedFields={selectedFields}
-              itemGetter={itemGetter}
-              onCancel={() => {
-                const itemsBeingEdited = new Set(value.itemsBeingEdited);
-                itemsBeingEdited.delete(id);
-                onChange({
-                  ...value,
-                  itemsBeingEdited,
-                });
-              }}
-            />
-          </CardContainer>
-        ) : (
-          <CardContainer mode="view" key={id}>
-            <Stack gap="xlarge">
-              {field.display.cardFields.map(fieldPath => {
-                const field = foreignList.fields[fieldPath];
-                const itemForField: Record<string, any> = {};
-                for (const graphqlField of getRootGraphQLFieldsFromFieldController(
-                  field.controller
-                )) {
-                  const fieldGetter = itemGetter.get(graphqlField);
-                  if (fieldGetter.errors) {
-                    const errorMessage = fieldGetter.errors[0].message;
-                    return (
-                      <FieldContainer>
-                        <FieldLabel>{field.label}</FieldLabel>
-                        {errorMessage}
-                      </FieldContainer>
-                    );
-                  }
-                  itemForField[graphqlField] = fieldGetter.data;
-                }
+      <Stack
```

Comment:
We only want to wrap the list items in a `ul`, the actions for creating a new item / inline connection to an existing item should sit outside of this list.